roles/taca/tasks/main.yml

```diff
@@ -69,6 +72,9 @@
 - name: Deploy sthlm sarek config
   template: src="site_taca_sarek_delivery.yml.j2" dest="{{ ngi_pipeline_conf }}/TACA/{{ site }}_taca_sarek_delivery.yml"
 
+- name: Deploy sthlm sarek WES config
+  template: src="site_taca_sarek_wes_delivery.yml.j2" dest="{{ ngi_pipeline_conf }}/TACA/{{ site }}_taca_sarek_wes_delivery.yml"
```

Comment:
Not sure if Sthlm will use this, so perhaps this task is not needed?

Reply:
I think we'll use this, but it may require some modification. We can address that in another PR maybe.